docs/user/supported_devices.rst

```diff
@@ -250,6 +250,7 @@ lantiq-xrx200
 
 * AVM
 
+  - FRITZ!Box 7360 (v1, v2, SL) [#avmflash]_
```

Comment:
Being picky here - The 7360SL is another model, while the v1, v2 denotes the revision.

Reply:
Done